### DISCUSSION TITLE
Welcome to pxMaxScript Discussions!

### DISCUSSION THREAD

**Discussion by piXelicidio** [piXelicidio/pxMaxScript#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gomelchuck34** [comment]
Hello,

I was using pixel paint for 3ds Max for an older version.  I am now running 3ds max 2024 and downloaded the ms script in github and run than script.  I then made created the toolbar button Poly Paint from customize user interface.   When I run the program I get a script error and can't get the tool to run.   Here is what I get.   Is there something I am doing wrong or do you have a fix for it?  It is an amazing tool and thanks for your time.


<img width="1220" height="749" alt="PolyPaint_error" src="https://github.com/user-attachments/assets/1e297e32-00d1-4970-9e99-5ac0eb72a102" />

**Reply by gomelchuck34** [reply]
I got it to work by downloading the .cs file and placing it in the ENU folder and renaming it

**Reply by piXelicidio** [reply]
Hi, I'm glad you found a quick fix. Sorry my delayed answer. I'll try to make more clear and practical the need for the c# code to be along with the MaxScript script.
